### PR TITLE
feat!: v2

### PR DESCRIPTION
**Pull request by domenicsim1**
feat: remove unused tags

### CODE REVIEW

File: __tests__/integration/integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    const projectGroup = (await repository.projectGroups.all())[0]
+    if (!projectGroup) throw new Error("can't find first projectGroup")
+
+    const project = await repository.projects.create({
+      Name: localProjectName,
+      LifecycleId: lifeCycle.Id,
+      ProjectGroupId: projectGroup.Id
+    })
+    standardInput.project = project.Id
+    globalCleanup.add(async () => repository.projects.del(project))
```
Comment (borland) [inline __tests__/integration/integration.test.ts:72]: the `del` method returns a Promise already, so should be able to remove the async

`globalCleanup.add(() => repository.projects.del(project))`

File: __tests__/integration/integration.test.ts [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+      DefaultGuidedFailureMode: GuidedFailureMode.EnvironmentDefault,
+      Name: 'TestRunbook',
+      EnvironmentScope: RunbookEnvironmentScope.All,
+      MultiTenancyMode: TenantedDeploymentMode.TenantedOrUntenanted,
+      RunRetentionPolicy: {
+        QuantityToKeep: 1,
+        ShouldKeepForever: false
+      }
+    })
+    globalCleanup.add(async () => repository.runbooks.del(runbook))
```
Comment (borland) [inline __tests__/integration/integration.test.ts:135]: as above can drop async lambda

File: __tests__/integration/integration.test.ts [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+    snapshot.RunbookId = runbook.Id
+    snapshot = await repository.runbookSnapshots.create(snapshot, {
+      publish: true
+    })
+    standardInput.runbook = runbook.Id
+    const env = await repository.environments.create({
+      Name: `Test-${runId}`
+    })
+    globalCleanup.add(async () => repository.environments.del(env))
+    globalCleanup.add(async () => new Promise(r => setTimeout(r, 2500)))
```
Comment (borland) [inline __tests__/integration/integration.test.ts:193]: ? why is there a 2.5 second sleep here? Also async lambda shouldn't be neccessary

File: __tests__/integration/integration.test.ts [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+    const messages: string[] = []
+    const w = new OctopusCliWrapper(
+      standardInput,
+      {},
+      m => messages.push(m),
+      m => messages.push(m)
+    )
+    await w.runRunbook(octoExecutable)
+    console.log('Got: ', messages)
+    // expectMatchAll(messages, [
```
Comment (borland) [inline __tests__/integration/integration.test.ts:217]: Suggest removing this commented-out code and replace with a comment explaining that the output is variable and we can't really assert on it

**Review by borland — CHANGES_REQUESTED** [review]
Minor comments, but I want to resolve why there's the 2500ms sleep in the integration test

**Review by jbristowe — APPROVED** [review]

**Review by borland — APPROVED** [review]

**Pull request merged**